servers/src/test/java/tachyon/web/WebUtilsTest.java

```diff
@@ -21,111 +21,112 @@
 import org.junit.Test;
 
 /**
- * Unit tests for {@link Utils}.
+ * Unit tests for {@link WebUtils}.
  */
-public class UtilsTest {
+public class WebUtilsTest {
```

Comment:
p.s., this is a convention in Tachyon codebase that if possible, always mark non-base-classes `final`.